server/src/main/resources/application.yaml

```diff
@@ -33,3 +34,13 @@ hedera:
     name: "$HEDERA_DATABASE_NAME:hedera"
     username: "$HEDERA_DATABASE_USERNAME:hedera"
     password: "$HEDERA_DATABASE_PASSWORD:hedera"
+
+  mail:
+    host: "$HEDERA_MAIL_HOST:127.0.0.1"
```

Comment:
I am not quite sure every single of those should have defaults, especially the "from" field. If it does, then a more sensible default than the current one should be found

Reply:
I would remove default for "from", "username" and "password". 

But maybe we could remove every default, so we can detect that mails are not configured and display a warning somewhere for the admins, as well as turning password reset off?

Reply:
the first idea is sufficient ; the second seems good

Reply:
Let's just do the first one and do the latest one another time.